AlinaFirsova/Homework 1/style.css

```diff
@@ -0,0 +1,19 @@
+front_main{ 
+    font-size: 100%; 
```

Comment:
@AlinaFirsova Please use approptiate font-size value, avoid of using percentage in fonts